Article.js

```diff
@@ -0,0 +1,32 @@
+class Article {
+  constructor(title,content,writer) {
+    this.title = title;

```

Comment:
생성자에서 현재 setter를 호출하는데, setter는 `this._title`에 저장합니다. 하지만 getter는 `this._title`을 반환하므로 항상 undefined가 됩니다.

```suggestion
    this._title = title;

```